types/ckeditor__ckeditor5-editor-inline/ckeditor__ckeditor5-editor-inline-tests.ts

```diff
@@ -1,4 +1,4 @@
-import { Editor } from '@ckeditor/ckeditor5-core';
```

Comment:
Is this really needed?

Reply:
Yeah, this is needed because the tests reference the `Plugin` symbol [here](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/a5e706fab1bc9a59c8797833becad4546f1b5be1/types/ckeditor__ckeditor5-editor-inline/ckeditor__ckeditor5-editor-inline-tests.ts#L34) and without this import, it implicitly references the browser global [Plugin](https://developer.mozilla.org/en-US/docs/Web/API/Plugin). I'm not quite sure why this wasn't a problem before, but probably something to do with the changes to the `PluginInterface` definition caused [this line](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/a5e706fab1bc9a59c8797833becad4546f1b5be1/types/ckeditor__ckeditor5-editor-inline/ckeditor__ckeditor5-editor-inline-tests.ts#L41) to start failing with:

![Fullscreen_8_5_21__7_34_AM](https://user-images.githubusercontent.com/559001/128368458-2bd67a17-efc3-4a5a-87d8-176e681c30df.jpg)

because `MyPlugin` was a `window.Plugin`, not a CKEditor `Plugin`.